flexmeasures/data/scripts/clean_database.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+cd /temp
```

Comment:
Should this not be `cd /tmp`?
I don't have `/temp` on my system ...

Also, I'd expect
- a backup to be made where I am currently.
- to be back where I was when this script is run (not in /tmp)

You can remember where you are now with `PWD=$(pwd)`